packages/calcite-components/src/components/accordion-item/accordion-item.scss

```
@@ -1,210 +1,202 @@
+/**
+ * CSS Custom Properties
+ *
+ * These properties can be overridden using the component's tag as selector.
+ *
+ * @prop --calcite-accordion-border-color: Specify the border color of the component
+ * @prop --calcite-accordion-text-color: Specify the text color of the component
+ * @prop --calcite-accordion-text-color-highlighted: Specify the highlighted text color
```

For the above doc consistency, we've used "Specifies" with other components, such as [Action](https://developers.arcgis.com/calcite-design-system/components/action/#styles) and periods should close out each description:

```suggestion
 * @prop --calcite-accordion-border-color: Specifies the border color of the component.
 * @prop --calcite-accordion-text-color: Specifies the text color of the component.
 * @prop --calcite-accordion-text-color-highlighted: Specifies the highlighted text color.
```